starfish/src/test/java/org/maxandersen/ExampleResourceTest.java

```diff
@@ -0,0 +1,17 @@
+package org.maxandersen;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+public class ExampleResourceTest {
+
+    @Test
+    public void testHelloEndpoint() {
```

Comment:
Add actual tests..

Reply:
I've currently no idea what to add in tests.
I am exploring  resources to learn more.
should there be any good/useful resource, please share with me